app/app.iml

```diff
@@ -102,39 +102,39 @@
     </content>
     <orderEntry type="jdk" jdkName="Android API 23 Platform" jdkType="Android SDK" />
```

Comment:
@jstevensog, simply don't call `git add` on that file. It will be created/updated by IntelliJ automatically when importing the project.